scripts/dump_meta_data.sql

```diff
@@ -18,8 +18,9 @@
 select version();
 
 \echo 'Build tag'
+\set ON_ERROR_STOP 0
```

Comment:
Is the output blocked here? Otherwise I believe this test will fail in some builds where git info is not available since the output depends on whether one builds in a Git repo with Git installed.

Reply:
AFACT, output is blocked here, but these lines are added because the test did not continue to execute after this statement.

Reply:
Tested in a separate run and they pass even when git is removed from the ABI tests and these lines are kept.

https://github.com/timescale/timescaledb/actions/runs/400648476